src/libAtomVM/module.h

```diff
@@ -114,7 +114,7 @@ struct Module
 
     const struct ExportedFunction **imported_funcs;
 
-    void **labels;
+    const uint8_t **labels;
```

Comment:
Since the goal of this PR is to improve performance might it be better to use `uint_fast8_t` here? I believe than even simple arithmetic is faster on esp32 and stm32 platforms using `uint16_t` vs `uint8_t`, so by using `uint_fast8_t` you can guarantee at least 8 bits, but use the optimal size for the platform.